todolist/settings.py

```diff
@@ -18,10 +18,10 @@
 # See https://docs.djangoproject.com/en/1.7/howto/deployment/checklist/
 
 # SECURITY WARNING: keep the secret key used in production secret!
-SECRET_KEY = "@e2(yx)v&tgh3_s=0yja-i!dpebxsz^dg47x)-k&kq_3zf*9e*"
+SECRET_KEY = os.getenv("SECRET_KEY", "default_secret_key")
```

Comment:
Using `os.getenv("SECRET_KEY", "default_secret_key")` is a good practice for development, but ensure that the `default_secret_key` is not used in production. Consider raising an error if the environment variable is not set in production environments.